### DISCUSSION TITLE
Using the new nextjs Image component with material-ui

### DISCUSSION THREAD

**Discussion by terchris** [vercel/next.js#19516] [Help]
In material-ui the components that display images have a parameter for the image. eg:
```
<Avatar alt="Remy Sharp" src="/static/images/avatar/1.jpg" />
```
In nextjs v 10 there is a new Image component that automatically scales images.  https://nextjs.org/docs/api-reference/next/image 
Have anyone figured out how to use the new Image component with material-ui.com ?

**Comment by jurajkrivda** [comment]
```
<Avatar variant="rounded" className={classes.rounded}>
  <NextImage />
</Avatar>
```
?

**Reply by StarAzure** [reply]
What is `<NextImage />` ?

nextjs docs show
`import Image from 'next/image'`

**Comment by ElyasMehraein** [comment]
<Avatar sx={{ width: 80, height: 80, ml: 4.75, mt: -2 }}>
            <Image
              alt="profile picture"
              src={UserAvatarImg}
              placeholder="blur"
              quality={100}
              fill
              sizes="100vw"
              style={{objectFit: 'cover',}}
            />
          </Avatar>
